### ISSUE TITLE
508-defect-2 [SCREEN READER]: Create a "results" slide upon ending the VR&E orientation

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22660]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement
This ticket seeks to resolve the following defect-level-2 issues related to the "Finish VR&E Orientation" button:
1. As a screen reader user, I expect the slideshow buttons to navigate in a consistent manner.
1. As a screen reader user, I expect headings to be logically ordered
1. As a screen reader user, I expect to hear what has been added to the page.
1. As a screen reader and keyboard user, I expect the "Finish VR&E Orientation" button to continue working even after the first use.
1. As a sighted and screen reader user, I expect action links that direct to the same page to share the same link text.

## Details

Upon clicking the "Finish VR&E Orientation" button, focus is sent to the action link "Apply for Veteran Readiness and Employment" which lives below and outside of the orientation grouping underneath the `h3` Independent Living Track. 

This interaction pattern introduces several issues that can be resolved with a results slide:

### 1. As a screen reader user, I expect the slideshow buttons to navigate in a consistent manner.
Since users have become accustomed with the slideshow navigation, adding new content to the page outside of the grouping instead of loading a final results slide would break navigational convention.

### 2. As a screen reader user, I expect headings to be logically ordered.
Currently, the newly added information lives below the `h3` Independent Living Track, which does not make sense in terms of a page outline. Loading a final results slide presents the opportunity to have a final `h3` explicitly titled "You've completed the VR&E Orientation" followed by additional information and the action link.

### 3. As a screen reader user, I expect to hear what has been added to the page.
The information above the action link, "Thank you for viewing the VR&E orientation. To apply for Veteran Readiness & Employment benefits now, click the button below." remains unannounced to screen reader users and is not logically discoverable. By having a specific results slide after clicking the "Finish VR&E Orientation" button, this information will be more intuitively discoverable.

### 4. As a screen reader and keyboard user, I expect the "Finish VR&E Orientation" button to continue working even after the first use.
The "Finish VR&E Orientation" button does not send focus to the action link after it has been activated once, which may cause confusion should a user navigate backwards to the button again and fail to receive additional feedback. Making the "Finish VR&E Orientation" button navigate in a similar manner to the previous/next slide buttons will not only improve consistency, but it will also resolve this one-time usage issue.

### 5. As a sighted and screen reader user, I expect action links that direct to the same page to share the same link text.
The action link in the wizard and the action link in the final slide share the same destination, but have different text. This may also appear in content feedback from @peggygannon . I recommend picking only one of the two, but will defer to @peggygannon on what is best.

## Acceptance Criteria

The proposed solution should achieve the following:

- [ ] Headings are logically ordered
- [ ] Newly added information is announced on the page (unless when handled within the slideshow which implicitly suggests navigation to a new slide)
- [ ] Buttons do not deactivate upon a single use
- [ ] Action links that share the same destination share the same link text

## Proposed Solution

<img width="1328" alt="Screen Shot 2021-04-05 at 4 02 59 PM" src="https://user-images.githubusercontent.com/14154792/113620699-aec5f800-9628-11eb-8627-77c8a5567273.png">

**Comment by joshkimux** [comment]
@jason-gcio @micahchiang This ticket may resolve @peggygannon 's feedback:

> After a user goes through the orientation, this alert appears:
> Thank you for viewing the VR&E orientation. To apply for Veteran Readiness & Employment benefits now, click the button below.
> {link} Apply for Veteran Readiness and Employment
> There is a link not a button below this content, please remove button reference from this alert.
> Also this link language differs from other link language that’s also on the wizard, which says "Apply online now with VA Form 28-1900"

Would love your thoughts here @peggygannon on whether or not this ticket addresses your feedback!

**Comment by jason-gcio** [comment]
Check and see if e2e test need to be updated?

**Comment by sporeboy** [comment]
Created mockup with my recommendations in place: https://xd.adobe.com/view/b8e1ab4c-4d8a-4f71-8d6c-00e4c1e809cc-edb2/screen/e3c93f70-0ccc-4202-8541-6f39354e97db

[Uploading... Screen Shot 2021-04-22 at 11.55.37 AM.png]
